app/Http/Controllers/Api/LicensesController.php

```diff
@@ -231,10 +231,6 @@ public function seats(Request $request, $licenseId)
             $offset = request('offset', 0);
             $limit = request('limit', 50);
 
-            if($seats->count() < $offset){
```

Comment:
Why was this removed?

Reply:
Lines 234 to 236 where a fix that I made first, then you recommended me that I better use the $total variable for the condition, and I added it later in the lines 237 to 239:

`if($total < $offset){
                $offset = 0;
            }`

But I forgot to remove the piece that compares $seats->count(). It doesn't have anything to do with the bug, but I revisited this code to recall how it works and then I realized.